CONTRIBUTING.md

```diff
@@ -37,3 +37,14 @@
 - Contributor License [Agreement](https://opensource.google/docs/cla/)
 - In [Solidarity](https://developers.google.com/style/inclusive-documentation)
 - [Pipelines](./PIPELINES.md)
+
+## Local lint runs
```

Comment:
I appreciate the nudge here. Ideally this would be a separate thread (PR) but I am OK with having it in here. 

My preferred way for contributors would be to enable actions to run on their fork and have the same actions run on their feature branch pushes as they will have on their PR.

Reply:
Happy to get rid of the additional note if it isn't the best option. 
A local instantaneous run of lint helps with productivity. The post merge checks are quality gates and they make sense as quality gates.

Reply:
Yeah if you don't mind. We intentionally wanted to leave the CONTRIBUTING as simple as possible.

If you want to include the links here: https://github.com/apigee/devrel/blob/main/PIPELINES.md#static-checks that would be a better place I think.

Reply:
Done